src/Stryker.Core/Stryker.Core.UnitTest/Options/Inputs/OptimizationModeInputTests.cs

```diff
@@ -0,0 +1,21 @@
+using Shouldly;
+using Stryker.Core.Exceptions;
+using Stryker.Core.Options;
+using Stryker.Core.Options.Inputs;
+using Xunit;
+
+namespace Stryker.Core.UnitTest.Options.Inputs
+{
+    public class OptimizationModeInputTests
+    {
+        [Fact]
+        public void ShouldValidateOptimisationMode()
```

Comment:
In this and similar places, maybe it's worth to also tests the "happy" path, like nothing bad happens in case of proper input.